### PR TITLE
feat(relayer): Support updating Relayer ubaClient

### PR DESCRIPTION
**Pull request by pxrl**
This is a PoC, there are outstanding points to resolve on the arguments to ubaClient.update() (i.e. for relayer looping mode).

Additionally, per current implementation, the ubaClient will update "TokensBridged" events. This will slightly increase RPC usage but should otherwise have no noticeable effect.

### CODE REVIEW

File: src/relayer/RelayerClientHelper.ts [hunk 1]
```diff
@@ -105,19 +106,26 @@ export async function constructRelayerClients(
...
-    "EnabledDepositRoute",
-    "RelayedRootBundle",
-    "ExecutedRelayerRefundRoot",
-  ]);
+  const { configStoreClient, spokePoolClients, ubaClient } = clients;
+
+  await configStoreClient.update();
+  const version = configStoreClient.getConfigStoreVersionForTimestamp();
+  if (sdkUtils.isUBA(version)) {
+    await ubaClient.update({}, true);
```
Comment (james-a-morris) [inline src/relayer/RelayerClientHelper.ts:114]: This is on me, but the update function should be called with `undefined` if you're not looking to override state

Comment (james-a-morris) [inline src/relayer/RelayerClientHelper.ts:114]:
```suggestion
    await ubaClient.update(undefined, true);
```

**Review by james-a-morris — COMMENTED** [review]
Left a comment

File: src/relayer/RelayerClientHelper.ts (hunk 1 above)
Comment (pxrl) [reply src/relayer/RelayerClientHelper.ts]: I've updated this PR to handle that, but requiring the caller to pass in `undefined` as part of normal operation is counter-intuitive for me.

**Review by nicholaspai — APPROVED** [review]

**Review by james-a-morris — APPROVED** [review]
LGTM

File: src/relayer/RelayerClientHelper.ts [hunk 2]
```diff
@@ -105,20 +107,27 @@ export async function constructRelayerClients(
...
-    "EnabledDepositRoute",
-    "RelayedRootBundle",
-    "ExecutedRelayerRefundRoot",
-  ]);
+  const { configStoreClient, spokePoolClients, ubaClient } = clients;
+
+  await configStoreClient.update();
+  const version = configStoreClient.getConfigStoreVersionForTimestamp();
+  if (sdkUtils.isUBA(version)) {
+    assert(configStoreClient.isValidConfigStoreVersion(version), `Relayer does not support UBA transfers`);
```
Comment (pxrl) [inline src/relayer/RelayerClientHelper.ts:115]:
```suggestion
    assert(configStoreClient.isValidConfigStoreVersion(version), "Relayer does not support UBA transfers");
```

**Pull request merged**